apps/src/signUpFlow/LoginTypeSelection.tsx

```diff
@@ -115,10 +116,10 @@ const LoginTypeSelection: React.FunctionComponent = () => {
               name="confirmPasswordInput"
             />
           </div>
-          <Button
+          <LinkButton
```

Comment:
Nit: I _think_ this can stay as a Button since it's submitting a form.

Reply:
Oh I guess it sort of does both? It will be both starting a sort-of-user session object but also redirecting the user to a new page (not sure which makes most sense for that so I'll keep it as `Button`)